### PR TITLE
feat(accordion): add compact and very compact variations

### PR DESCRIPTION
**Pull request by prudho**
## Description
This PR adds `compact` and `very compact` variations for the `accordion` module.

## Screenshot (if possible)
image

### CODE REVIEW

**Review by exoego — DISMISSED** [review]
LGTM

File: src/definitions/modules/accordion.less [hunk 1]
```diff
@@ -176,6 +176,56 @@
   }
 }
 
+& when (@variationAccordionCompact) {
+  /*--------------
+       Compact
+  ---------------*/
+
+  /* Default Styling */
+  .ui.compact.accordion:not(.styled) .title:not(.ui) {
```
Comment (lubber-de) [inline src/definitions/modules/accordion.less:185]: Why is `:not(.ui)` necessary? We don't have a `ui title` or `ui content` selector? What am i missing?

Comment (prudho) [reply src/definitions/modules/accordion.less:185]: I was trying to wonder why too. Seems it's already set in initial declaration:
```css
/* Default Styling */
.ui.accordion .title:not(.ui) {
  padding: @titlePadding;
  font-family: @titleFont;
  font-size: @titleFontSize;
  color: @titleColor;
}

/* Default Styling */
.ui.accordion:not(.styled) .title ~ .content:not(.ui),
.ui.accordion:not(.styled) .accordion .title ~ .content:not(.ui) {
  margin: @contentMargin;
  padding: @contentPadding;
}
.ui.accordion:not(.styled) .title ~ .content:not(.ui):last-child {
  padding-bottom: 0;
}
```

Maybe an old remnant code from v1 🤔

Comment (lubber-de) [reply src/definitions/modules/accordion.less:185]: Can you try to remove that `:not(.ui)` from everywhere in accordion.less ? I searched the whole source , there definately isn't any `.ui.*.content` or title selector, so i strongly believe we can removed that completely.

Comment (prudho) [reply src/definitions/modules/accordion.less]: Done by `af3ed42`.

I also doubt that `.ui.accordion .accordion` is still used nowadays.

**Review by lubber-de — APPROVED** [review]
LGTM

**Comment by lubber-de** [comment]
Docs added by https://github.com/fomantic/Fomantic-UI-Docs/pull/334

**Pull request merged**